### ISSUE TITLE
Pretrained weight problem

### ISSUE THREAD

**Issue by MianHtan** [blindwatch/IINet#4]
I used the pretained weight to test the sceneflow test set, it only get epe 0.57 (different from paper 0.54). Is that a different checkporint or is the problem of my cuda and torch version.
My environment: cuda11.8, torch2.3.1.

**Comment by blindwatch** [comment]
It is the same checkpoint as that used in the paper. I checked locally before uploading. Since the gap between 0.57 and 0.54 seem to be small, it may caused by version difference between torch 2.x and torch 1.x.

**Comment by MianHtan** [comment]
Thank you for your answer! I can get the same result after using the same version of pytorch.
